kolibri/plugins/learn/assets/src/vue/breadcrumbs/breadcrumb/index.vue

```diff
@@ -1,11 +1,9 @@
 <template>
 
   <a v-link="linkobject">
-    <span class='back'>
-      <span v-if="showarrow" class="visuallyhidden">{{ $tr('back') }}</span>
-      <svg v-if="showarrow" role="presentation" src="../../icons/back.svg"></svg>
-      {{ text }}
-    </span>
+    <span v-if="showarrow" class="visuallyhidden">{{ $tr('back') }}</span>
```

Comment:
judge from the class name visuallyhidden, this span may serve accessibility purpose. I will add the `$trs: back` back